homeassistant/components/homewizard/config_flow.py

```diff
@@ -305,65 +305,65 @@ async def async_step_reconfigure(
             errors=errors,
         )
 
-    @staticmethod
-    async def _async_try_connect(ip_address: str) -> Device:
-        """Try to connect.
 
-        Make connection with device to test the connection
-        and to get info for unique_id.
-        """
+async def async_try_connect(ip_address: str) -> Device:
```

Comment:
Moved this from `HomeWizardConfigFlow._async_try_connect` to make it accessible by repairs.py. No code changes.